tests/converters/goldbeck_test.py

```diff
@@ -0,0 +1,31 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from unittest.mock import Mock
+
+import pytest
+from openpyxl.reader.excel import load_workbook
+from parkapi_sources.converters import GoldbeckPushConverter
+
+from tests.converters.helper import get_data_path, validate_static_parking_site_inputs
+
+
+@pytest.fixture
+def goldbeck_push_converter(mocked_config_helper: Mock) -> GoldbeckPushConverter:
+    return GoldbeckPushConverter(config_helper=mocked_config_helper)
+
+
+class GoldbeckPushConverterTest:
+    @staticmethod
+    def test_get_static_parking_sites(goldbeck_push_converter: GoldbeckPushConverter):
+        workbook = load_workbook(filename=str(get_data_path('goldbeck.xlsx').absolute()))
+
+        static_parking_site_inputs, import_parking_site_exceptions = goldbeck_push_converter.handle_xlsx(workbook)
+
+        assert len(static_parking_site_inputs) > len(
```

Comment:
After some more validation, it turned out that it makes sense to check for actual lengths, in order to make sure that it's not just a few more successes then failures, and to look at the failures closely. See https://github.com/ParkenDD/parkapi-sources-v3/blob/9f662f218c99aa62063ac2407f2e79395521b407/tests/converters/apcoa_test.py#L54 , for example.

Reply:
Ok i have added the assertion validation.